test/domainRegicter.js

```diff
@@ -0,0 +1,119 @@
+import { expect } from "chai";
```

Comment:
each test block should be independent from each other
as good practice to use describe block for each test
// - 1 point